cmake/Git.cmake

```diff
@@ -0,0 +1,31 @@
+#=============================================================================
+# Copyright 2011 Kitware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
It probably came from ITK. Brad imported it back in 2011 but doesn't mention where it came from.